### DISCUSSION TITLE
How can I send a message to a new phone number?

### DISCUSSION THREAD

**Discussion by idoaflalo** [Auties00/WhatsappWeb4j#96] [Q&A]
How can I send a message to a new phone number (which I never sent a message before)?

**Comment by diegofreitas** [comment]
You can create an contact object and send the message like this.
ContactJid contactJid = ContactJid.builder().server(ContactJid.Server.WHATSAPP).user(user).build();
api.sendMessage(contactJid,  "This is a text message send by cobalt wp api!");

where user is the phone number without "+" sigal

**Reply by geckel123** [reply]
Is there any alternative to this code in the current version of the library?

**Comment by lucioufba** [comment]
Hi @diegofreitas , Is there a way to check if a phone number is a WhatsApp contact?
